src/app/tests/suites/TestFabricSyncBridgedNode.yaml

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Test Fabric Synchronization condition on Bridged Node Device Type
+
+config:
+    nodeId: 0x12344321
+
+tests:
+    - label:
+          "Read the DeviceTypeList attribute of the Descriptor cluster and
+          check whether the device type is Aggregator."
+      endpoint: 1
+      cluster: "Descriptor"
+      command: "readAttribute"
+      attribute: "DeviceTypeList"
+      response:
+          value: [{
+              DeviceType: 14, # Aggregator
+          }]
+
+    - label:
+          "Read the ServerList attribute of the Descriptor cluster and check
+          whether the Commissioner Control is present."
+      endpoint: 0
+      cluster: "Descriptor"
+      command: "readAttribute"
+      attribute: "ServerList"
+      response:
+          constraints:
+              type: list
+              contains: [
+                  0x0751, # Commissioner Control Cluster
+              ]
+
+    - label:
+          "Read the SupportedDeviceCategories attribute of the Commissioner
+          Control cluster and check whether the FabricSynchronization bit is
+          set."
+      endpoint: 0
```

Comment:
Is the commissioner controller cluster mandated to be on ep0?

Reply:
Right now it is a little ambiguous if this is supposed to be on the root endpoint or aggregator endpoint. Right now example app assumed root endpoint.

This is one of those things were we are going to assume root node for the 29th and get clarification on spec and update accordingly

Reply:
I've added dedictaed config for that, so once the spec will be clarified and the app fixed, one can change the  Commissioner Control cluster number